BT-Tracking/Interface/Active/ActiveAppController.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ActiveAppController.swift
+//  BT-Tracking
+//
+//  Created by Jakub Skořepa on 20/03/2020.
+//  Copyright © 2020 Covid19CZ. All rights reserved.
+//
+
+import UIKit
+
+
+class ActiveAppController: UIViewController {
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(applicationDidBecomeActive),
+            name: UIApplication.didBecomeActiveNotification,
+            object: nil
+        )
+        
+        checkForBluetooth()
+    }
+    
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        
+        NotificationCenter.default.removeObserver(
+            self,
+            name: UIApplication.didBecomeActiveNotification,
+            object: nil
+        )
+    }
+    
+    @IBOutlet private weak var shareButton: UIButton!
+
+    @IBAction func shareApp() {
+        let url = URL(string: "https://www.google.com")!
+        let shareContent = [url]
+        let activityViewController = UIActivityViewController(activityItems: shareContent, applicationActivities: nil)
+        activityViewController.popoverPresentationController?.sourceView = shareButton
+        
+        present(activityViewController, animated: true, completion: nil)
+    }
+    
+    @objc private func applicationDidBecomeActive() {
+        checkForBluetooth()
+    }
+    
+    private func checkForBluetooth() {
+        if (UIApplication.shared.delegate as? AppDelegate)?.scanner.isRunning != true {
```

Comment:
Dal jsem tam peknou vunkci jak se dostat k app delegate "AppDelegate.shared"

 a radeji mam "," misto "&&" :)

Reply:
It's `AppDelegate.delegate`, `AppDelegate.shared` or `UIApplication.delegate` would be better :) 
I've used it

Reply:
It's `AppDelegate.delegate`, `AppDelegate.shared` or `UIApplication.delegate` would be better :) 
I've used it